.env.example

```
@@ -4,4 +4,19 @@ MAINNET_DEPLOYER_NAME=
 SEPOLIA_DEPLOYER_NAME=
 
+BASE_RPC=
+ARBITRUM_RPC=
+GNOSIS_RPC=
+CELO_RPC=
+OPTIMISM_RPC=
+POLYGON_RPC=
+
+# For upgrade:submit-to-safe (proposer must be a Safe owner)
+SAFE_ADDRESS=
+PROXY_ADMIN_ADDRESS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PROXY_ADMIN_ADDRESS key should go before the SAFE_ADDRESS key

(UnorderedKey)